_docs/tasks/fault-injection.md

```diff
@@ -21,12 +21,15 @@ This task shows how to inject delays and test the resiliency of your application
 * Initialize the application version routing by either first doing the
   [request routing](./request-routing.html) task or by running following
   commands:
+  
+  > Note: This assumes you don't have any default routes set yet. If you've already created default route rules for the sample, you'll need to use `replace` rather than `create` in the following commands.
```

Comment:
It's not just setting "default" rules in these commands, so I think it should reworded slightly:

Note: This assumes you don't have any routes set yet. If you've already created conflicting route rules for the sample, you'll need to use `replace` rather than `create` in one or both of the following commands.